packages/babel-core/src/transformation/file/index.js

```diff
@@ -432,7 +432,13 @@ export default class File extends Store {
 
       const loc = err.loc;
       if (loc) {
-        err.codeFrame = codeFrame(code, loc.line, loc.column + 1, this.opts);
+        const location = {
+          start: {
+            line: loc.line,
+            column: loc.column + 1,
```

Comment:
We don't have any tests for it probably so I didn't really check, dono if you want to take a look